.circleci/config.yml

```diff
@@ -14,11 +14,14 @@ workflows:
               only: /v?[0-9]+(\.[0-9]+)*/
           matrix:
             parameters:
-              python_version: ["2.7", "3.5"]
-              django_version: ["django111", "django22"]
```

Comment:
@andrey-canon are you going to add the tests for Django30,31 in a different PR?

Reply:
not, that is out of the scope